macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -161,6 +175,9 @@ open class AvatarView: NSView {
 	/// The layer used to mask the overall AvatarView to a circle
 	private let circleMask = CAShapeLayer()
 
+	/// The apperance observer to update colors when theme is changed
+	private var appearanceObserver: NSKeyValueObservation?
```

Comment:
I think if we go with this approach we also need to invalidate this token on deinit. But also, could we just do the color calculation inside updateAppearance() in  updateLayer()? (I know I told you to go for this method of KVO, sorry). I'm gonna summon @anandrajeswaran because he had strong opinions about this the last time I spoke with him

Reply:
`viewWillMove` is called anytime there is a change in layout hierarchy. It does set appearance to nil on line 420